core/src/test/java/org/testcontainers/utility/DockerImageNameCompatibilityTest.java

```diff
@@ -91,6 +91,12 @@ public void testAssertMethodAcceptsCompatible() {
         subject.assertCompatibleWith(DockerImageName.parse("bar"));
     }
 
+    @Test
+    public void testAssertMethodAcceptsCompatibleLibraryPrefix() {
```

Comment:
Suggestion:
```
public static Stream<Arguments> methodSource() {
  return Stream.of(
    Arguments.of(DockerImageName.parse("foo").asCompatibleSubstituteFor("bar"), DockerImageName.parse("bar")),
    Arguments.of(DockerImageName.parse("library/foo"), DockerImageName.parse("foo")),
  );
}


@ParameterizedTest
@MethodSource("methodSource")
public void testAssertMethodAcceptsCompatibleLibraryPrefix(DockerImageName subject, DockerImageName expectation) {
   subject.assertCompatibleWith(expectation);
}
```

Reply:
thanks for the suggestion but we are not using junit jupiter

Reply:
https://docs.openrewrite.org/